packages/bits/angular.json

```diff
@@ -184,7 +184,7 @@
                         "main": "",
                         "namedChunks": true,
                         "outputPath": "./.tmp-css",
-                        "sourceMap": true,
+                        "sourceMap": false,
```

Comment:
Why are you changing this to `false` sourceMaps are a good way to debug in the inspector.